### ISSUE TITLE
Error when doing anything with soundcloud

### ISSUE THREAD

**Issue by Anthys** [TeamNewPipe/NewPipe#2745]
- [ X] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [X ] I checked if the issue/feature exists in the latest version.
- [X ] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

Hi, this happens whenever I open the Soundcloud part of the app, and doesn't let me do anything. 

## Exception
* __User Action:__ requested kiosk
* __Request:__ https://soundcloud.com/charts/top
* __Content Language:__ GB
* __Service:__ SoundCloud
* __Version:__ 0.17.3

<details><summary><b>Crash log</b></summary><p>

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.jsoup.nodes.Element.attr(java.lang.String)' on a null object reference
	at org.schabi.newpipe.extractor.services.soundcloud.SoundcloudParsingHelper.clientId(SoundcloudParsingHelper.java:49)
	at org.schabi.newpipe.extractor.services.soundcloud.SoundcloudChartsExtractor.computNextPageAndStreams(SoundcloudChartsExtractor.java:56)
	at org.schabi.newpipe.extractor.services.soundcloud.SoundcloudChartsExtractor.getInitialPage(SoundcloudChartsExtractor.java:85)
	at org.schabi.newpipe.extractor.utils.ExtractorHelper.getItemsPageOrLogError(ExtractorHelper.java:20)
	at org.schabi.newpipe.extractor.kiosk.KioskInfo.getInfo(KioskInfo.java:69)
	at org.schabi.newpipe.extractor.kiosk.KioskInfo.getInfo(KioskInfo.java:55)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getKioskInfo$10(ExtractorHelper.java:170)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$TneUC77tEl15twhVz-_2huMZ3Ck.call(Unknown Source:4)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.Maybe.subscribe(Maybe.java:4154)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.Flowable.subscribe(Flowable.java:14479)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.Maybe.subscribe(Maybe.java:4154)
	at io.reactivex.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.processTask(ThreadPoolExecutor.java:1187)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:784)

```
</p></details>
<hr>

**Comment by daemonspudguy** [comment]
Same issue

**Comment by TobiGr** [comment]
Discussion will be in TeamNewPipe/NewPipeExtractor#205

**Comment by ildar** [comment]
Nice! when the next release is planned?

**Comment by TobiGr** [comment]
Fixed by https://github.com/TeamNewPipe/NewPipeExtractor/pull/209
Next release should be out shortly.

**Comment by ADepic** [comment]
Wasn't fixed in 0.18.0.
See #2915 for more info
